src/views/CrontabForm/CrontabForm.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from "react";
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  ActionGroup,
+  Button,
+  PageSection,
+  Title,
+  Alert,
+  Spinner,
+} from "@patternfly/react-core";
+import { CRONTAB_APIGROUP, CRONTAB_APIVERSION, CRONTAB_KIND } from "src/const";
+import {
+  useK8sModel,
+  k8sCreate,
+  useActiveNamespace,
+} from "@openshift-console/dynamic-plugin-sdk";
+import { useHistory } from "react-router-dom";
+import { useNavigate } from "react-router-dom-v5-compat";
+import { useCronTabTranslation } from "@crontab-utils/hooks/useCronTabTranslation";
+
+export const CronTabForm: React.FC = () => {
+  const [model] = useK8sModel({
+    group: CRONTAB_APIGROUP,
+    version: CRONTAB_APIVERSION,
+    kind: CRONTAB_KIND,
+  });
+  const [name, setName] = useState("");
+  const [cronSpec, setCronSpec] = useState("");
+  const [image, setImage] = useState("");
+  const [replicas, setReplicas] = useState<number | string>(1); // Initialize as 1 or an empty string
```

Comment:
Parse the replicas input as an integer before setting it so that this always a number:
```suggestion
  const [replicas, setReplicas] = useState(1);
```